app/helpers/users_helper.rb

```diff
@@ -16,14 +16,7 @@ module UsersHelper
   #   gravatar.
   #
   def gravatar_for(user, options = {})
-    options = {
-      size: 48
-    }.merge(options)
-
-    size = options[:size]
-    gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
-    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
-    image_tag(gravatar_url, alt: user.name, class: 'gravatar')
+    ENV['DISABLE_GRAVATAR'].present? ? no_gravatar_image : gravatar_image(user, options)
```

Comment:
They get set up by an environment variable: `FEATURES=a,b,c,d,...` In omnibus they get defined here: https://github.com/chef/omnibus-supermarket/blob/a1c159ce0793287b154a9edad1473ad75e762ed5/cookbooks/omnibus-supermarket/attributes/default.rb#L388-L403